Documentation/Development.md

```diff
@@ -121,3 +120,17 @@ $ Utilities/bootstrap --generate-xcodeproj --enable-perf-tests
 ```
 
 Then, open the generated project and run the `PerformanceTest` scheme.
+
+## Testing on Linux with Docker
+
+For contributors on macOS who need to test on Linux, install Docker and use the
+following commands:
+
+```sh
+$ Utilities/docker build # will build an image with the latest swift snapshot
+$ Utilities/docker boostrap # will bootstrap SwiftPM on the linux container
+$ Utilities/docker run bash # to run an interactive bash sheel in the container
+$ Utilities/docker swift-build # to run swift-build in the container
+$ Utilities/docker swift-test # to run swift-test in the container
+$ Utilities/docker swift-run # to run swift-run in the container
+```
```

Comment:
I would recommend not calling the script docker, just to avoid the alias.

Reply:
Renamed to `docker-utils`.